__tests__/__data__/config/custom-types.json

```diff
@@ -8,8 +8,9 @@
       "added": "New Feature",
       "changed": "Enhancement",
       "deprecated": "Soon Removing",
-      "removed": "Deleted",
+      "feature": "Feature",
```

Comment:
"Feature" is added here primarily to help test that things are sorted properly. We want to ensure that this entirely custom type is not at the bottom of the list.